docs/recipe_09.rst

```diff
@@ -0,0 +1,314 @@
+    1         -1.52917957    0.41925484    1.99037051
+    1         -3.54789329   -1.53307223   -0.78323817
+    1         -1.52137470   -1.28387237   -2.89754915
+    *
+
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+Reading Simple ORCA Output Files
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+Reading an ORCA output file is as simple as::
+
+  path = 'test/static/orca_uridine_opt_freq.out'
+  file = cctk.OrcaFile.read_file(path)
+
+From the resulting ORCA file we can acccess several properties that apply to the whole job::
+  
+  file.job_types
+
+returns a list of job types::
+
+  [<OrcaJobType.OPT: 'opt'>, <OrcaJobType.FREQ: 'freq'>, <OrcaJobType.SP: 'sp'>]
+
+indicating the job contains an opt and freq calculations. A single point calculation is included in all jobs::
+
+  file.successful_terminations
+  # returns '3' confirming successful termination of all three job types
+
+  file.num_imaginaries()
+  # returns '0', confirming that there are no imaginary frequencies in the optimzied structure
+
+We can also access the full ensemble of geometries from the geometry optimization and their corresponding properties::
+
+  path = 'test/static/orca_uridine_opt_freq.out'
+  file = cctk.OrcaFile.read_file(path)
+
+  file.ensemble.properties_list()
+  # returns a list of dictionaries
+  # each dictionary in the list corresponds to a geometry from the optimization
+  # each dictionary contains property keys mapped to property values for the specified geometry.
+
+  file.ensemble.get_properties_list()[0]
+
+Returns the properties dictionary of the first geometry in the ensemble::
+
+  # {'energy': -911.078699594417,
+  # 'filename': './test/static/orca_uridine_opt_freq.out',
+  # 'iteration': 0,
+  # 'scf_iterations': 13.0,
+  # 'rms_gradient': 0.004391566,
+  # 'max_gradient': 0.0237232031,
+  # 'rms_step': 0.0198286671,
+  # 'max_step': 0.067347534}
+
+To access the frequencies from the last geometry in the ensemble::
+
+  freqs = file.ensemble[-1,'frequencies']
+  # assigns a list of frequencies to the variable freqs
+
+To access a given property for each member of the ensemble::
+
+  geom_iters = file.ensemble[:,'iteration']
+  energy = file.ensemble[:, 'energy']
+  rms_grad = file.ensemble[:, 'rms_gradient']
+
+
+We can then easily plot the property as a function of optimization step:: 
+
+  import matplotlib.pyplot as plt
+
+  f1 = plt.figure(figsize=(8,6))
+  plt.scatter(geom_iters, energy)
+  plt.ylabel(f"energy (hartree)")
+  plt.xlabel(f"geometry step")
+  plt.close()
+
+  f2 = plt.figure(figsize=(8,6))
+  plt.scatter(geom_iters, rms_gradient)
+  plt.ylabel(f"rms_gradient")
+  plt.xlabel(f"geometry step")
+  plt.close()
+
+Calling ``f1`` returns:
+
+.. image:: ./img/r09_step_vs_energy.png
+    :width: 450
+    :align: center
+
+Calling ``f2`` returns:
+
+.. image:: ./img/r09_step_vs_rms_grad.png
+    :width: 450
+    :align: center
+
+
+""""""""""""""""""""""""""""""""
```

Comment:
No need to include these here as they are already in the repo.  You could just add links.